prime-router/src/main/kotlin/Report.kt

```diff
@@ -117,12 +117,14 @@ class Report {
         values: List<List<String>>,
         sources: List<Source>,
         destination: OrganizationService? = null,
+        bodyFormat: Format? = null
     ) {
         this.id = UUID.randomUUID()
         this.schema = schema
         this.sources = sources
         this.createdDateTime = OffsetDateTime.now()
         this.destination = destination
+        this.bodyFormat = bodyFormat ?: destination?.format ?: Format.INTERNAL
         this.table = createTable(schema, values)
```

Comment:
Typically, it will be the destination format, but it can be set to different value.